frontend/backend-types.ts

```diff
@@ -140,10 +140,6 @@ export interface paths {
      */
     post: operations["issueCredentialConnectionLess"];
   };
-  "/api/issuer/issue-credential/oob-attachment/{id}": {
```

Comment:
The endpoint above is the one used by the UI, this one I hid because it should never be called directly, only when it is referenced in the invitation url.

Reply:
I see, my bad :+1: